data-visualization-of-the-week-template.ipynb

```diff
@@ -22,6 +22,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "### Instructions:\n",
+    "#### Step 1 (your only step): “Run” the cells to see the graphs\n",
+    "Click “Cell” and select “Run All.” This will import the data and run all the code to make this weeks data visualization (scroll down after you’ve run the cells).  **You don’t need to do any coding if you don't want to**.\n",
```

Comment:
usually when you click Cell and Run All, it will leave you at the bottom of the notebook. So I think your instructions should say 
`"Click “Cell” and select “Run All.” This will import the data and run all the code to make this week's data visualizations (scroll to the top after you’ve run the cells).  **You don’t need to do any coding**.\n",`

Reply:
Cell now reads:

` "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Instructions:\n",
    "#### Step 1 (your only step): “Run” the cells to see the graphs\n",
    "Click “Cell” and select “Run All.” This will import the data and run all the code to make this week's data visualizations (scroll to the top after you’ve run the cells). **You don’t need to do any coding**.\n",
    "\n",
    "![instructions](images/instructions.png)"
   ]
  },`